docker-compose.yml

```diff
@@ -2,7 +2,12 @@ version: '3.9'
 
 services:
   tests:
+    image: my-php-app
```

Comment:
There's no need to add this.

Reply:
@stnguyen90 I have applied the changes suggested by you. The only problem I am facing right now is the lint error caused by those unrelated files. What should I do about it? Composer formatting fixes it by deleting the comments. But is that acceptable?